substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2597,6 +2655,7 @@ mod benches {
 		[tasks_example, TasksExample]
 		[pallet_democracy, Democracy]
 		[pallet_asset_conversion, AssetConversion]
+		[pallet_asset_rewards, AssetRewards]
```

Comment:
Freezer as well?

Reply:
it actually do not have calls, only fungibles trait impls